### PR TITLE
LPS-188648 [MARKETPLACE] Implement Replace For DDM Template

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by nNilton** [comment]
What about the new ddm templates which are going to be created during site initialization? How can they be replaced in layouts? @joaodamasceno2001

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 1]
```diff
@@ -947,6 +951,7 @@ private void _addFragmentEntries(
 	private void _addLayoutContent(
 			Map<String, String> assetListEntryIdsStringUtilReplaceValues,
 			Map<String, String> clientExtensionEntryIdsStringUtilReplaceValues,
+			Map<String, String> ddmTemplateIdsStringUtilReplaceValues,
 			Map<String, String> ddmStructureEntryIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:955]: sort

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 2]
```diff
@@ -1834,7 +1858,14 @@ private void _addOrUpdateDDMTemplates(
 					SiteInitializerUtil.read(_bundle, "ddm-template.ftl", url),
 					false, false, null, null, serviceContext);
 			}
+
+			ddmTemplateIdsStringUtilReplaceValues.put(
+				"DDMTEMPLATE_ID:" +
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1863]:
```suggestion
				"DDM_TEMPLATE_ID:" +
```

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Pull request closed without merging**